app/views/static/home.html.erb

```diff
@@ -1,11 +1,187 @@
-Erudition
-</br>
-</br>
-<% if !user_signed_in? %>
-  <%= link_to 'Sign up', new_user_registration_path %>
-  </br>
-  <%= link_to 'Log in', new_user_session_path %>
-<% end %>
-<% if user_signed_in? %>
-  <%= link_to 'Log out', destroy_user_session_path, method: :delete %>
-<% end %>
+<div class="page page-home">
+
+  <nav class="navbar navbar-default">
+    <div class="container">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="/">Erudition</a>
+      </div>
+      <div class="navbar-header navbar-right">
+        <% if !user_signed_in? %>
+          <%= link_to 'Log in', new_user_session_path, class: 'btn btn-default navbar-btn nav-pills' %>
+        <% end %>
+        <% if user_signed_in? %>
+          <p class="navbar-text">Signed in as email@email.com</p>
+          <%= link_to 'Log out', destroy_user_session_path, method: :delete, class: 'btn btn-default navbar-btn' %>
+        <% end %>
+      </div>
+    </div>
+  </nav>
```

Comment:
Since the `nav` will be used across the application, you'll probably want to take the `nav` element out of the `page-home` div and make it a partial. From there you can render the partial in the application layout, allowing you to maintain the code in only one place as well as only having to render it in one place. Take a look here for more info http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials